nmmo/core/observation.py

```diff
@@ -87,11 +88,13 @@ def tile(self, r_delta, c_delta):
         Vector corresponding to the specified tile
     '''
     agent = self.agent()
+    center = self.config.PLAYER_VISION_RADIUS
+    tile_dim = self.config.PLAYER_VISION_DIAMETER
+    mat_map = self.tiles[:,2].reshape(tile_dim,tile_dim)
     if (0 <= agent.row + r_delta < self.config.MAP_SIZE) & \
        (0 <= agent.col + c_delta < self.config.MAP_SIZE):
-      r_cond = (self.tiles[:,TileState.State.attr_name_to_col["row"]] == agent.row + r_delta)
-      c_cond = (self.tiles[:,TileState.State.attr_name_to_col["col"]] == agent.col + c_delta)
-      return TileState.parse_array(self.tiles[r_cond & c_cond][0])
+      return SimpleNamespace(**{'row': agent.row+r_delta, 'col': agent.col+c_delta,
```

Comment:
Are you saying it's faster to construct a SimpleNamespace instead of TileState?

Reply:
no difference, so I'm using parse_array again